apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -4,28 +4,42 @@
  * External dependencies
  */
 import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+import './style.scss';
 
 domReady( () => {
 	const { closeButtonLabel, closeButtonUrl, editorPostType } = fullSiteEditing;
 
 	const editPostHeaderInception = setInterval( () => {
-		const closeButton = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar a' );
+		// cycle through interval until header toolbar is found
+		const toolbar = document.querySelector( '.edit-post-header__toolbar' );
 
-		if ( ! closeButton ) {
+		if ( ! toolbar ) {
 			return;
 		}
 		clearInterval( editPostHeaderInception );
 
+		// add components toolbar with override class name (original will be hidden in ./style.scss)
+		const componentsToolbar = document.createElement( 'div' );
+		componentsToolbar.className =
+			'components-toolbar edit-post-fullscreen-mode-close__toolbar edit-post-fullscreen-mode-close__toolbar__override';
+		toolbar.prepend( componentsToolbar );
+
+		// create custom close button and append to components toolbar
+		const newCloseButton = document.createElement( 'a' );
 		// When closing Template CPT (e.g. header) to navigate back to parent page.
 		if ( 'wp_template_part' === editorPostType && closeButtonUrl ) {
-			const newCloseButton = document.createElement( 'a' );
 			newCloseButton.href = closeButtonUrl;
 			newCloseButton.innerHTML = closeButtonLabel;
-			newCloseButton.className = 'components-button components-icon-button is-button is-default';
 			newCloseButton.setAttribute( 'aria-label', closeButtonLabel );
-
-			const parentContainer = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar' );
-			parentContainer.replaceChild( newCloseButton, closeButton );
+		} else if ( 'page' === editorPostType ) {
```

Comment:
cc @shaunandrews @olaolusoga (btw who should I CC for design review?): This would change the button from an arrow icon to "Back to Page List" for the page editor in the full screen editor as well. Just want to make sure this is a fine change. (it's consistent with the "back to page" button in the template editor)

Reply:
> Since this makes sure the button always exists, it works towards #36610 as well. However, since the button is so wide, it makes things look weird on mobile:

We could make this phrase "Back to Pages" instead of "Back to Page List" in the meantime.  Cutting off those last 4 characters does seem to help this for most modern phone screen sizes.

Reply:
Hm, yeah. One thing that might be strange in this context is that it is possible to land on this screen without coming from the page list. For example, if you're coming from the checklist, or from the "customize site" link after theme activation, the back button ought to take you somewhere else. (Sometimes this is handled by Calypso), so the static label might not make sense all of the time.

Reply:
(Which I think might be partially why we hadn't added it to the page editor yet)

Reply:
>btw who should I CC for design review?
CC'ing both of us for now is fine.

>Just want to make sure this is a fine change.

Can we keep the back arrow icon, and add "Pages" copy to the right of it, vs a button. Though I get the button exists somewhere else, it breaks the toolbar pattern. If we're trying to provide guidance to users on how to get back to pages, adding text to the right of the icon for context should be enough.

Mock of back icon with copy:
<img width="453" alt="Screen Shot 2019-10-31 at 9 33 31 PM" src="https://user-images.githubusercontent.com/1382943/67996601-654cce80-fc26-11e9-86d9-cf8fda19c0ae.png">

Reply:
> We could make this phrase "Back to Pages" instead of "Back to Page List" in the meantime. Cutting off those last 4 characters does seem to help this for most modern phone screen sizes.

Just pointing out that we should still be mindful of other languages that might be longer anyway.

Reply:
@olaolusoga - Definitely!  I think that looks great with the arrow and short label as you suggest.  I have updated the code to do it this way.  Thank you. Having this kind of guidance for design is greatly appreciated.  =)

For mobile view this seems to be a good size fit for most modern screen widths.  However it does still cause some issues on thin phones such as iPhone 5.  I have gone ahead and made the "Pages" label disappear on screen widths <= 360px.  Hopefully this leaves enough room to account for other languages as @Copons suggests, but if we see an issue we can hide this label starting at a larger value.

I think this resolves @noahtallen 's complaint as "Pages" seems to be where that button currently leads no matter how we get to the page editor, but "back to" does not always make sense.